x-pack/plugin/security/qa/jwt-realm/src/javaRestTest/java/org/elasticsearch/xpack/security/authc/jwt/JwtRestIT.java

```diff
@@ -114,6 +120,7 @@ public class JwtRestIT extends ESRestTestCase {
         .keystore("xpack.security.authc.realms.jwt.jwt2.hmac_key", HMAC_PASSPHRASE)
         .keystore("xpack.security.authc.realms.jwt.jwt3.hmac_jwkset", HMAC_JWKSET)
         .keystore("xpack.security.http.ssl.secure_key_passphrase", "http-password")
+        .keystore("xpack.security.transport.ssl.secure_key_passphrase", "http-password")
```

Comment:
nit: This should ideally be a different password from the http passphrase

Reply:
Totally, addressing this.